compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SemanticAnalyzer.java

```diff
@@ -2280,6 +2280,11 @@ public void visit(BLangForever foreverStatement) {
 
     @Override
     public void visit(BLangConstant constant) {
+        Name constantName = names.fromIdNode(constant.name);
```

Comment:
We use this variable only in L2284 right? 

How about we define a variable for `constant.name` instead, since that seems to be used several times.